arbeitszeit_flask/templates/macros/tooltips.html

```diff
@@ -2,9 +2,9 @@
 <div class="dropdown is-right">
     <div class="dropdown-trigger">
         <span class="icon" aria-haspopup="true" aria-controls="{{ id }}">
-            <i class="fa-solid fa-info-circle">
+            {{ "info-circle"|icon }}
```

Comment:
I think there's something wrong, isn't it? In chrome and firefox it shows like this:
![image](https://github.com/user-attachments/assets/ccceb50c-5ef8-4ae4-b170-b50c8f04a686)

Reply:
Thanks for noticing this! It went under in the super repetitive text search and replacement frenzy I had with all those icons 😉 I can directly spot the reason for this tooltip layout issue. I will fix it and come back to you.